src/constants.js

```diff
@@ -0,0 +1,20 @@
+const CONSTANTS = {
+  field: {
+    integer: new RegExp(/^-?\d+$/),
+    money: new RegExp(/^([A-Z]{3}) (-?\d+)$/),
+  },
+
+  header: {
+    sku: 'variant-sku',
+  },
+
+  standardOption: {
+    batchSize: 100,
+    delimiter: ',',
+    strictMode: true,
+  },
+}
+
+Object.freeze(CONSTANTS)
```

Comment:
@Siilwyn For this to be significant, you might want to freeze the subobjects too. Check this [out](http://stackoverflow.com/a/33124316/4151025)